flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/debezium/DebeziumUtils.java

```diff
@@ -120,6 +122,39 @@ public static BinlogOffset earliestBinlogOffset(JdbcConnection jdbc) {
         }
     }
 
+    public static BinlogOffset seekBinlogOffsetByTimestamp(String binlogFile, BinlogOffset maxBinlogOffset,
+                                                           MySqlSourceConfig sourceConfig, long timestamp) {
+        BinaryLogClient client = new BinaryLogClient(sourceConfig.getHostname(), sourceConfig.getPort(),
+                sourceConfig.getDatabaseList().get(0), sourceConfig.getUsername(), sourceConfig.getPassword());
+        client.setServerId(sourceConfig.getServerIdRange().getEndServerId());
```

Comment:
while ServerIdRange is null , start from timestamp will occurs error, maybe we can throws an exception?